.travis.yml

```diff
@@ -21,5 +21,11 @@ before_script:
   - git config --global user.email team+camperbot@freeCodeCamp.org
   - git config --global user.name "CamperBot"
 
-script:
-  - npm run test
+jobs:
+  include:
+    - stage: Test the things
+      script: npm test
+
+    - stage: Trigger build
+      if: branch = master && type != pull_request
```

Comment:
We should have a issue tracking "CI Stage for production"

Reply:
In my head, I have something that builds the api image, runs the image and queries the api to ensure the correct responses are given.